### ISSUE TITLE
lib/vmsish.t: blocker to strict-by-default

### ISSUE THREAD

**Issue by jkeenan** [atoomic/perl#234]
As of commit ac6b7ccf73 in the alpha-dev-02-strict branch on Aug 18 2020, the file cited is failing.  Here is output from `cd t; ./perl harness -v <file>; cd -`:
```
# ../lib/vmsish.t

Global symbol "$Orig_Bits" requires explicit package name (did you forget to declare "my $Orig_Bits"?) at ../lib/vmsish.t line 20.
BEGIN not safe after errors--compilation aborted at ../lib/vmsish.t line 20.
# Looks like your test exited with 255 before it could output anything.
../lib/vmsish.t .. 
1..29
Dubious, test returned 255 (wstat 65280, 0xff00)
Failed 29/29 subtests 

Test Summary Report
-------------------
../lib/vmsish.t (Wstat: 65280 Tests: 0 Failed: 0)
  Non-zero exit status: 255
  Parse errors: Bad plan.  You planned 29 tests but ran 0.
Files=1, Tests=0,  0 wallclock secs ( 0.01 usr  0.00 sys +  0.04 cusr  0.00 csys =  0.05 CPU)
Result: FAIL
```

@atoomic @toddr 

Thank you very much.
Jim Keenan

**Comment by jkeenan** [comment]
This should be fixed by 3b3d50f043.  While only one test in this file will run on non-VMS platforms, we can at least get the program to compile under strict-by-default and run to completion.

I believe this ticket can be closed.

Thank you very much.
Jim Keenan

**Comment by jkeenan** [comment]
> This should be fixed by [3b3d50f](https://github.com/atoomic/perl/commit/3b3d50f043f59da03eca571f494f8d448c86ac43). While only one test in this file will run on non-VMS platforms, we can at least get the program to compile under strict-by-default and run to completion.
> 
> I believe this ticket can be closed.

Hearing no objection, closing this ticket.
